APIs/UpcomingChanges.qmd

```diff
@@ -0,0 +1,100 @@
+# Upcoming changes
+
+This is the list of upcoming changes to Catalog APIs.
+
+## OpenSearch interface error handling update
+
+Please be informed that the **OData API** interface will be updated soon. The exact date of the update will be announced in a separate release note no later than 5 days before the scheduled change.
+
+To be compliant with the OData PRIP interfaces the API response for a single product query will be updated. The response will now return the product itself instead of a list containing the product (assigned to the field `value`). The example reflecting the change is presented below.
```

Comment:
```suggestion
To comply with the OData PRIP interfaces, the API response for a single product query will be updated. The response will now return the product itself instead of a list containing the product (assigned to the field `value`). The example reflecting the change is presented below:
```